tests/valid_login.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test('login with valid credential', async ({ page }) => {
+    await page.goto('https://practicesoftwaretesting.com/auth/login');
+    await page.locator('[data-test="email"]').fill('customer@practicesoftwaretesting.com')
```

Comment:
Consider adding **testIdAttribute** to your config, that way, you will not have to use `locator()` with CSS selector, but - `getByTestId` method instead - https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute
```suggestion
    await page.getByTestId('"email").fill('customer@practicesoftwaretesting.com');
```